src/main/scala/org/neo4j/spark/Neo4jQuery.scala

```diff
@@ -0,0 +1,27 @@
+package org.neo4j.spark
+
+import org.neo4j.cypherdsl.core.Cypher
+import org.neo4j.cypherdsl.core.renderer.Renderer
+import collection.JavaConverters._
+
+object Neo4jQuery {
+
+  import QueryType._
+
+  val NODE_ALIAS = "n"
+  val INTERNAL_ID_FIELD = "<id>"
+  val INTERNAL_LABELS_FIELD = "<labels>"
+
+  private val renderer: Renderer = Renderer.getDefaultRenderer
+
+  def build(queryOptions: Neo4jQueryOptions): String =
+    queryOptions.queryType match {
+      case LABELS =>
+        val labels: List[String] = queryOptions.value.split(":").toList
```

Comment:
Labels are specified like this:   `Foo:Bar`.   That's good, looks very "neo4j-y".   When doing this and interpolating into queries though, remember that labels can have weird characters, and may be enclosed in backticks.  By making this a string, it encourages users to treat it like cypher.  

But what if I set labels to be  "\`Foo Bar\`:Baz" will this work?   As a user, I'd expect it to, since that's how I did it if I have a MATCH query.